Src/BootCamp.Chapter/CredentialsManager.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.IO;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    internal class CredentialsManager
+    {
+        public struct Person
```

Comment:
This is not a person. A person is not name and password. This is credentials.

Reply:
Changed the name